File: LeetCode/Easy/0598-range-addition-ii/0598-range-addition-ii.java
class Solution {
    public int maxCount(int m, int n, int[][] ops) {
        int x = m;
        int y = n;
        
        for (int i = 0; i < ops.length; ++i){
            if (ops[i][0] < x) x = ops[i][0];
            if (ops[i][1] < y) y = ops[i][1];
        }
        
        return x * y;
    }
}

File: LeetCode/Easy/0598-range-addition-ii/0598-range-addition-ii.go
func maxCount(m int, n int, ops [][]int) int {
    x := m
    y := n
    
    for i := 0; i < len(ops); i++ {
        if ops[i][0] < x { x = ops[i][0] }
        if ops[i][1] < y { y = ops[i][1] }
    }
    
    return x * y
}